R/dictionaries.R

```diff
@@ -93,7 +93,7 @@ split_values <- function(dict, concatenator_dictionary, concatenator_tokens) {
                                                  concatenator_tokens))
         names(result) <- key
     }
-    result
+    return(result)
```

Comment:
Makes no functional difference, however according to [our own style guide](https://github.com/quanteda/quanteda/wiki/Style-guide) we [should not use `return()`](https://style.tidyverse.org/functions.html#return) to send an object's evaluation back to the parent environment at the end of a function. I'm not bothered by it either way.

Reply:
I consider this one of the most strange R conventions. We should return values explicitly like in other (proper) programing languages.  For me, [Google R Style Guide](https://google.github.io/styleguide/Rguide.xml) is much more agreeable than **tidyverse**'s.

Reply:
They're about 90% compatible but the **tidyverse** guide is a bit more complete. We can't agree with https://google.github.io/styleguide/Rguide.xml#object for **quanteda** at least. (Sometimes with R you have to embrace a slight bit of chaos!)

We already have local exceptions (to both), so let's add the suggestion of an explicit `return(finalresult)` to the **quanteda** Style Guide.